sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -179,6 +179,7 @@ case class Expand(
 trait GroupingAnalytics extends UnaryNode {
   self: Product =>
   def gid: AttributeReference
+  def groupings: Seq[AttributeReference]
```

Comment:
Because the syntax `GROUPING (column name)` is SQL99.
https://msdn.microsoft.com/en-us/library/ms178544.aspx